integration-tests/smoke/automation_test.go

```diff
@@ -104,13 +103,13 @@ ListenAddresses = ["0.0.0.0:6690"]`
 	}
 )
 
-func TestMain(m *testing.M) {
-	logging.Init()
-	fmt.Printf("Running Smoke Test on %s\n", networks.SelectedNetwork.Name) // Print to get around disabled logging
-	fmt.Printf("Chainlink Image %s\n", os.Getenv("CHAINLINK_IMAGE"))        // Print to get around disabled logging
-	fmt.Printf("Chainlink Version %s\n", os.Getenv("CHAINLINK_VERSION"))    // Print to get around disabled logging
-	os.Exit(m.Run())
-}
+// func TestMain(m *testing.M) {
```

Comment:
@lukaszcl why is this part commented out?

Reply:
@anirudhwarrier good catch. It was just for testing purposes. I uncommented it